docs/CodeStyle.md

```diff
@@ -32,6 +32,14 @@
 ## C++
   * used only for unit testing
   * lower-case class names (same as stl/boost)
+  * the unit tests in test/gtest are written using [C++11](https://en.cppreference.com/w/cpp/11). Whenever applicable, the usage of advanced language features is allowed and preferred over legacy code. For example:
+    * Prefer references over pointers
+    * `auto` for type deduction 
+    * Use [move semantics](https://www.cprogramming.com/c++11/rvalue-references-and-move-semantics-in-c++11.html) where applicable
```

Comment:
minor: looks like other lines in this file starts with lower case

Reply:
I would change all to uppercase

Reply:
Yeah I noticed that.
Strictly speaking, all lines shall start with uppercase in all lines. The problem is that all lists in other parts of this file also start with lowercase.
Therefore, we either change the whole file in this PR (unrelated), or end up with one list inconsistent with others.

Reply:
imo, consistency is better. I'd either use lower case or change all lines, but it is up to you

Reply:
Capitalized first letter in each line.